skyportal/enum_types.py

```diff
@@ -1,16 +1,44 @@
-import sqlalchemy as sa
-from enum import Enum
+import astropy.units as u
 import inspect
+import numpy as np
+import sncosmo
+from sncosmo.bandpasses import _BANDPASSES
+from sncosmo.magsystems import _MAGSYSTEMS
+import sqlalchemy as sa
 
+from enum import Enum
 from baselayer.app.env import load_env
-
+from baselayer.log import make_log
 from . import facility_apis
 
-from sncosmo.bandpasses import _BANDPASSES
-from sncosmo.magsystems import _MAGSYSTEMS
+log = make_log('enum_types')
 
 _, cfg = load_env()
 
+# load additional bandpasses into the SN comso registry
+existing_bandpasses_names = [val['name'] for val in _BANDPASSES.get_loaders_metadata()]
+additional_bandpasses_names = []
+for additional_bandpasses in cfg.get('additional_bandpasses', []):
+    name = additional_bandpasses.get("name")
+    if not name:
```

Comment:
Where do you expect to encounter this?  I would expect this if you used `.get("name", None)` above.

Reply:
if it's a malformed entry (`name: ` or missing `name`), I dont want to throw an uncaught error.